oci/spec_opts_unix.go

```diff
@@ -1,5 +1,3 @@
-// +build !windows
```

Comment:
Rename the file to remove the `unix` suffix?

Reply:
I was trying to keep it mostly the same but you are right. Fixed.